### DISCUSSION TITLE
Laravel Octane : lambda timeout ?

### DISCUSSION THREAD

**Discussion by LemarinelNet** [brefphp/bref#1572] [Support]
Hi,

In the default lambda function setup, we set timeout to 28s .
Is there any reason we should change this setting to an upper limit with Octane ? 
Can/Should I change it to i.e. 720s since Octane would have benefits of running for longer ?

**Comment by semsphy** [comment]
If you are using **lambda URL**, You can increase the timeout to 720 seconds, but make sure it is not over 15 minutes (lambda timeout). The previous timeout (28s) was for the API gateway timeout.

**Comment by mnapoli** [comment]
Hi! Just to clarify, the timeout is for a single HTTP request! Not for the Octane server.

So the Octane server/process can run for hours/days. Does that make sense?

**Comment by LemarinelNet** [comment]
Ok !  I got it ;)
